src/pages/GamerPassport/ActionModals.tsx

```diff
@@ -0,0 +1,343 @@
+import React, { useState } from "react";
+import { faThumbsUp, faHeartCrack } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useWallet } from "@solana/wallet-adapter-react";
+import axios from "axios";
+import { itheumSolPreaccess } from "libs/sol/SolViewData";
+import { getApiWeb2Apps } from "libs/utils";
+import { ActionButton } from "./SharedComps";
+
+export const ActionModalStep1 = (props: any) => {
+  const { showActionModel, handleHideActionModel } = props;
+  const { handleStep1Passed, handleStep1PSNUserName, handleStep1EligibilityCheckResults } = props;
+
+  const [psnUserName, setPsnUserName] = useState<string>("");
+  const [eligibilityCheckLoading, setEligibilityCheckLoading] = useState<boolean>(false);
+  const [eligibilityCheckStepDone, setEligibilityCheckStepDone] = useState<boolean>(false);
+  const [eligibilityUserNameTaken, setEligibilityUserNameTaken] = useState<boolean>(false);
+  const [eligibilityUserNameNotSuitable, setEligibilityUserNameNotSuitable] = useState<boolean>(false);
+
+  async function checkEligibilityViaAPI() {
+    setEligibilityCheckLoading(true);
+
+    const checkRes = await axios.get(`${getApiWeb2Apps()}/datadexapi/gamepassport/checkEligibility/${psnUserName}`);
+    const eligibilityDecision = checkRes.data;
+
+    console.log(eligibilityDecision);
```

Comment:
Delete console log

Reply:
needed during initial stage of release for debugging